pkg/server/workspacetemplates/service_test.go

```diff
@@ -118,15 +118,17 @@ func (s *WorkspaceTemplateServiceTestSuite) SetupTest() {
 	s.workspaceTemplateStore = workspacetemplate_internal.NewInMemoryWorkspaceTemplateStore()
 	s.workspaceTemplateService = workspacetemplates.NewWorkspaceTemplateService(workspacetemplates.WorkspaceTemplateServiceConfig{
 		ConfigStore: s.workspaceTemplateStore,
-		FindNewestBuild: func(ctx context.Context, prebuildId string) (*models.Build, error) {
-			return s.buildService.Find(&stores.BuildFilter{
-				PrebuildIds: &[]string{prebuildId},
-				GetNewest:   util.Pointer(true),
+		FindNewestBuild: func(ctx context.Context, prebuildId string) (*services.BuildDTO, error) {
+			return s.buildService.Find(&services.BuildFilter{
+				StoreFilter: stores.BuildFilter{
+					PrebuildIds: &[]string{prebuildId},
+					GetNewest:   util.Pointer(true),
+				},
 			})
 		},
-		ListPublishedBuilds: func(ctx context.Context) ([]*models.Build, error) {
-			return s.buildService.List(&stores.BuildFilter{
-				States: &[]models.BuildState{models.BuildStatePublished},
+		ListPublishedBuilds: func(ctx context.Context) ([]*services.BuildDTO, error) {
```

Comment:
Maybe we can rename this to `ListSuccessfulBuilds` since we don't have the Published state anymore